projects/plugins/social/src/class-note.php

```diff
@@ -196,11 +209,37 @@ public function maybe_flush_rewrite_rules( $force = false ) {
 	 * @param array $post_id The Post ID.
 	 */
 	public function override_empty_title( $title, $post_id ) {
-		if ( get_post_type( $post_id ) === self::JETPACK_SOCIAL_NOTE_CPT ) {
-			return wp_trim_words( get_the_excerpt(), 10 );
+		$post = get_post( $post_id );
+
+		if (
+			$post instanceof \WP_Post &&
+			self::JETPACK_SOCIAL_NOTE_CPT === $post->post_type
+		) {
+			$publishing_date = new \DateTimeImmutable(
+				$post->post_date,
+				wp_timezone()
+			);
+
+			$title = sprintf(
+				/* Translators: placeholder is a fully-formatted date. */
+				__( 'Social note, %1$s', 'jetpack-social' ),
+				wp_date(
+					get_option( 'date_format' ) . ' \a\t ' . get_option( 'time_format' ),
```

Comment:
Looking back at my previous suggestion, I'm thinking the date format should be translatable, since `\a\t` doesn't make sense in languages other than English.

I think you could add something like this:

```suggestion
			$datetime_format = sprintf(
				/* Translators: %1$s is a formatted date, e.g. June 18, 2025. %2$s is a formatted time, e.g. 8:24 am. All other words/letters need to be escaped. */
				__( '%1$s \a\t %2$s', 'jetpack-social' ),
				get_option( 'date_format' ),
				get_option( 'time_format' )
			);

			$title = sprintf(
				/* Translators: placeholder is a fully-formatted date. */
				__( 'Social note, %1$s', 'jetpack-social' ),
				wp_date(
					$datetime_format,
```

Reply:
@jeherve It makes sense. FYI, I've applied the changes. Thanks!